data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
Yes.
Even if your last craft was the same as the suspended one, it just restarts. In order to resume, you need to manually select the same craft and it doesn't seem to always work.